.evergreen/evergreen.yml

```diff
@@ -134,12 +134,13 @@ buildvariants:
 
   - name: generator
     display_name: Generator
-    allowed_requesters: ["github_pr"]
+    patch_only: true
```

Comment:
Am confused as to how the generator task still ran on this PR 🤔 we do not want to limit the regular generator to patch_only, right?

Reply:
I think that's because the GitHub PR is technically a patch.

Reply:
`patch_only` includes CLI patches (`patch_request`), GitHub patches (`github_pull_request`), and GitHub merge patches (`github_merge_request`)